examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -591,16 +593,22 @@ def compute_metrics(eval_preds):
 
     if optim_args.apply_pruning:
 
-        if optim_args.end_epoch is None:
-            end_epoch = training_args.num_train_epochs
+        if optim_args.end_step is None:
+            end_step = training_args.num_train_epochs * (

```

Comment:
Shouldn't we include the number of `gradient_accumulation_steps` ? Also I think the number of epoch might be easier for users to set, what do you think ?

Reply:
Pruning occurs after the every optimizer.step() is called. So I think needn't include gradient_accumulation_steps. Our think the step can slightly adjust sparsity.